Makefile

```diff
@@ -36,6 +36,10 @@ install:
 kubeone: build
 build: dist/kubeone
 
+.PHONY: docker
+docker: dist/kubeone
+	docker build -t quay.io/kubermatic/kubeone:$(GITTAG) -f build/docker/Dockerfile .
```

Comment:
The `$(GITTAG)` is going to be something like `v0.9.0-1-<hash>` I believe. Or that's at least what I get when I run the command sourced by variable.

Do we want to use that or just version like `0.9.0`?

Reply:
I am unsure about that. Do we want some sort of "nightly" image? or do we want to build it on releases?